frontend/src/App.tsx

```diff
@@ -217,14 +221,40 @@ function App(props: { setOnDisconnect: Dispatch<SetStateAction<Callback | undefi
   }, [dispatchAppUpdate, setOnDisconnect]);
 
   const page = useMemo(() => {
-    if (!appState.sessionToken) {
-      return <Login doLogin={setupGameController} />;
-    } if (!videoInstance) {
-      return <div>Loading...</div>;
-    }
+    // TODO: This is only temporary. This needs to be removed
+    console.log("DEBUG: REMOVE ME PLEASE!")
+    // if (!appState.sessionToken) {
+    //   return <Login doLogin={setupGameController} />;
+    // } if (!videoInstance) {
+    //   return <div>Loading...</div>;
+    // }
```

Comment:
I've commented this block out temporarily so that we can directly reach the maps page for quicker development time (rather than go the select-towns page first and then going into the maps page). Since our entire frontend is literally just in the maps page, I wanted to persist this till the end of the project (we can uncomment this TODO after we are sure that our frontend work is done).